api/all/src/main/resources/META-INF/native-image/io.opentelemetry/opentelemetry-api/reflect-config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "resources": {
+    "includes": [
+      {
+        "pattern":"\\Qio/opentelemetry/api/version.properties\\E"
```

Comment:
What are the "\\Q" and "\\E" characters?

Reply:
The `pattern` property expects a regex: https://www.graalvm.org/latest/reference-manual/native-image/dynamic-features/Resources/

"\Q" and "\E"  are used to define  a literal string in the regex.

The GraalVM tracing agent has been used to generate this pattern value.